### PR TITLE
i18n: document translation administrators

### PR DESCRIPTION
**Pull request by dachary**
## Status

Ready for review

## Description of Changes

document translation administrators

## Testing

N/A

## Deployment

N/A

### If you made changes to documentation:

- [x] Doc linting passed locally

### CODE REVIEW

**Comment by dachary** [comment]
@msheiny that's a new one for you to enjoy :-) https://circleci.com/gh/freedomofpress/securedrop/6682?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

<pre>
    TASK [Add temporary SSH key to AWS region.] ************************************
    Wednesday 17 January 2018  09:56:39 +0000 (0:00:01.214)       0:00:14.457 *****
    An exception occurred during task execution. To see the full traceback, use -vvv. The error was: <Response><Errors><Error><Code>RequestLimitExceeded</Code><Message>Request limit exceeded.</Message></Error></Errors><RequestID>7652a309-1b7f-488b-8f48-396c2f8d1e99</RequestID></Response>
    fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "module_stderr": "Traceback (most recent call last):\n  File \"/tmp/ansible_lrL81p/ansible_module_ec2_key.py\", line 267, in <module>\n    main()\n  File \"/tmp/ansible_lrL81p/ansible_module_ec2_key.py\", line 155, in main\n    key = ec2.get_key_pair(name)\n  File \"/usr/local/lib/python2.7/dist-packages/boto/ec2/connection.py\", line 2854, in get_key_pair\n    dry_run=dry_run\n  File \"/usr/local/lib/python2.7/dist-packages/boto/ec2/connection.py\", line 2836, in get_all_key_pairs\n    [('item', KeyPair)], verb='POST')\n  File \"/usr/local/lib/python2.7/dist-packages/boto/connection.py\", line 1170, in get_list\n    response = self.make_request(action, params, path, verb)\n  File \"/usr/local/lib/python2.7/dist-packages/boto/connection.py\", line 1116, in make_request\n    return self._mexe(http_request)\n  File \"/usr/local/lib/python2.7/dist-packages/boto/connection.py\", line 1028, in _mexe\n    raise BotoServerError(response.status, response.reason, body)\nboto.exception.BotoServerError: BotoServerError: 503 Service Unavailable\n<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Response><Errors><Error><Code>RequestLimitExceeded</Code><Message>Request limit exceeded.</Message></Error></Errors><RequestID>7652a309-1b7f-488b-8f48-396c2f8d1e99</RequestID></Response>\n", "module_stdout": "", "msg": "MODULE FAILURE", "rc": 0}
</pre>

**Comment by msheiny** [comment]
`RequestLimitExceededRequest limit exceeded` ? WTF. thanks for the ping @dachary

File: docs/development/i18n.rst [hunk 1]
```diff
@@ -297,6 +297,54 @@ We do not want to publish the translator emails so we strip them:
...
+
+An administrator who has not be active for two months or more is
+removed from the list and their permissions revoked. They can apply
+again at anytime.
+
+The community of SecureDrop translators works very closely with the
+SecureDrop developers and some of them participate in both
+groups. However, the translators community has a different set of
+rules and permissions, reason why it makes sense to have an
+independant policy.
```
Comment (redshiftzero) [inline docs/development/i18n.rst]: `independant` -> `independent`

**Comment by dachary** [comment]
Thanks for the review and the reminder about the docs- branch, I'm getting better at it but still not consistent :-)

**Comment by dachary** [comment]
rebased & repushed !

**Pull request merged**